release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -762,6 +762,13 @@ OVS remains unable to dynamically use isolated CPUs assigned to containers in `G
 When the Node Tuning Operator recognizes the performance conditions to activate the use of non-reserved CPUs, there is a several second delay while OVN-Kubernetes configures the CPU affinity alignment of OVS daemons running on the CPUs. During this window, if a `Guaranteed` QoS pod starts, it can experience a latency spike.
 ====
 
+[id="ocp-4-14-dual-stack-configuration"]
+==== Dual stack configuration for multiple IP addresses
+
+In previous releases of Whereabouts IPAM CNI, only one IP address could be assigned per network interface. 
```

Comment:
```suggestion
In previous releases of the Whereabouts IPAM CNI plugin, only one IP address could be assigned per network interface. 
```
Adding a little more context here for any readers who might not immediately know that Whereabouts is an IPAM CNI plugin.